client/app/routes/dashboards/dashboard.js

```diff
@@ -424,9 +424,11 @@ export default Ember.Route.extend({
                     {
                         chartType: 'donut',
                         widgetType: 'c3-chart',
-                        name: 'Publishers',
+                        name: 'Data providers',
```

Comment:
Please change "Publishers" to "Data providers" here as well so that the chart shows the right data: https://github.com/cos-labs/share-analytics/blob/master/client/app/components/c3-chart/component.js#L66